CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ add_subdirectory(GX-common)
 add_subdirectory(GX-TimeFrequency)
 
 # Documentation
-option(ENABLE_DOCS "Enable documentation" OFF)
+option(ENABLE_DOCS "Enable documentation" ON)
```

Comment:
This should stay off as default, else doxygen becomes a mandatory requirement for building the lib.

I already updated the docs to say default is to not build the docs

Reply:
Ok.